.drone.yml

```diff
@@ -10,16 +10,19 @@ trigger:
 
 steps:
   - name: setup
-    image: abakus/lego-testbase:python3.11
+    image: abakus/lego-testbase:latest
     when:
       event: push
     environment:
       LANG: C.UTF-8
+      CACHE: cache
```

Comment:
what does this do?

Reply:
Not sure yet. I was just testing because the cache was behaving weirdly.